dt/uni01alpha/kustomization.yaml

```diff
@@ -189,6 +189,18 @@ replacements:
         options:
           create: true
```

Comment:
Also would need to get openstack-operator updated for new octavia api.

Also let's test this before merging after all the fixes.

Reply:
agreed